queries/usersQueries.js

```diff
@@ -1,15 +1,21 @@
-const usersData = require('../usersData.json');
+// const usersData = require('../usersData.json');
+const db = require('../db/index');
 
-const idToUserIndex = usersData.reduce((index, user) => {
-  const { id } = user;
-  index[id] = user;
-  return index;
-}, {});
+// const idToUserIndex = usersData.reduce((index, user) => {
+//   const { id } = user;
+//   index[id] = user;
+//   return index;
+// }, {});
```

Comment:
You should delete this code before you commit it. A lot of teams have rules against committing commented out code, and it can make your project code look bad to interviewers.